src/main/webapp/WEB-INF/themes/mobile/ftl/article/relatedContent.ftl

```diff
@@ -23,6 +23,7 @@
 <#include "../baseTemplates/articleSection.ftl" />
 <#assign mainclass = "related-content" />
 <#assign mainId = "content" />
+<#assign mainClass = "content" />
```

Comment:
These `mainClass` assignments have turned out to be pretty important.  Can you help me understand what caused us to need to add these?  I'm guessing an html change of `content` from an id to a class?